src/store/reducers/iotHubManagerReducer.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Name this file devicesReducer instead of referencing the service. It may be that the devices reducer ends up calling multiple services so there isn't a 1-to-1 mapping from reducers and services (or even reducers to pages).

Reply:
OK. That makes sense.  I'll learn how things interact better soon, I'm sure.  For now, I was going with the 1 to 1 mapping.